django-stubs/db/models/query.pyi

```diff
@@ -194,7 +194,12 @@ class QuerySet(Generic[_Model, _Row], Iterable[_Row], Sized):
     def __getitem__(self, s: slice) -> Self: ...
 
 class RawQuerySet(Iterable[_Model], Sized):
+    raw_query: RawQuery | str
+    model: type[_Model] | None
```

Comment:
should `model` in `__init__` be also updated?

Reply:
yep, done